tasking/pulp/tasking/scheduler.py

```diff
@@ -258,7 +258,7 @@ def tick(self):
                           % {'celerybeat_name': self.celerybeat_name})
             ret = self.call_tick(self, self.celerybeat_name)
         except TaskLock.DoesNotExist:
-            TaskLock.objects.get(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
+            TaskLock.objects.filter(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
```

Comment:
Yeah, it looks like  it is not needed here.
But... Is it possible that it was an attempt to delete any old celerybeat lock (from other machine/node as well) which for some reason was not cleaned up properly before? 
In such case the lock should be specified and not the name:
`TaskLock.objects.filter(lock=TaskLock.CELERY_BEAT, timestamp__lte=old_timestamp).delete()`

@bmbouter , @dkliban what do you think?